src/bthread/bthread.h

```diff
@@ -26,7 +26,7 @@
 
 #if defined(__cplusplus)
 #  include <iostream>
-#  include "bthread/mutex.h"        // use bthread_mutex_t in the RAII way
+#  include "bthread/mtx_cv_base.h"        // use bthread_mutex_t in the RAII way
```

Comment:
Please explain why the name is changed

Reply:
This one is not just a rename. We need to separate low-leveled facilities like `Mutex` and `ConditonVariable` into separate headers so that other higher-leveled c++ constructs can depend on them. Otherwise there will be circular dependencies.
mtx_cv_base.h is never meant to be included directly by users.